Docs: Improve Okta docs (#89453)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md

1. Click **Save**.
1. Click the **Back to applications** link at the top of the page.
1. From the **More** button dropdown menu, click **Refresh Application Data**.

> 1. Ensure the `groups` scope is included in the `Scopes` (`scopes` in Terraform and config file) configuration of the Okta integration.

Reviewer: This lacks the extra explanation that was there originally:

> a. Extend the `scopes` field of `[auth.okta]` section in Grafana configuration file with the refresh token scope used by your OIDC provider.

What's the reason for that change?

Author: I removed that part because it was partly correct, copied from the Generic OAuth documentation ("with the refresh token scope used by your OIDC provider."). And it was redundant, because the `Configure a refresh token` section has all the required information + it contain the required scope name ("offline_access").

Reviewer: But this section doesn't link to that, isn't "Extend the scopes field of `[auth.okta]` section in Grafana configuration file with the refresh token scope used by your OIDC provider." more descriptive than "(`scopes` in Terraform and config file) configuration of the Okta integration."? Or are they talking about different things and I've only linked them by the word `scopes`.

Author: > Or are they talking about different things and I've only linked them by the word scopes.

I guess yes, the "Ensure the `groups` scope is included..." part is added to make sure they have `groups` in the list of `scopes`. The other part was removed because it was redundant, but that is unrelated to this change.

> "Extend... scope used by your OIDC provider." more descriptive than

It's more descriptive, but we know that Okta uses the `offline_access` scope and it's been documented in the `Configure a refresh token` section.

Reviewer: Gotcha, I don't think being more explicit hurts. Let's tell the reader exactly what to add where.

I don't see any other reference to "`Scopes`" (capitalized) in this page. Where would someone see that word, is it part of a different configuration?

Should the parenthesized section of this sentence actually be the key information like in:

```suggestion
1. Include the `groups` scope in the value for the `scopes` field of the Okta integration.
```
Are there more options than just Terraform or Grafana configuration file?

Author: > I don't see any other reference to "Scopes" (capitalized) in this page. Where would someone see that word, is it part of a different configuration? + Are there more options than just Terraform or Grafana configuration file?

Yes, `Scopes` refer to the label of the input box on the UI.

Reviewer: Ah in that case it should be in bold: https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold.

Perhaps:

```suggestion
1. Include the `groups` scope in the **Scopes** field in Grafana of the Okta integration.
   For Terraform or in the Grafana configuration file, include the `groups` scope in `scopes` field.
```
